README.md

```diff
@@ -66,7 +66,16 @@ as well as it clones and analyzes all relevant repositories. If you need to upda
 the number,
 
 1. ask him to run the script,
-2. bother him to Open Source the script code.
+1. bother him to Open Source the script code.
+
+### Google Drive credentials
+
+1.  Follow the steps in the [gspread guide](https://gspread.readthedocs.io/en/latest/oauth2.html). Instead of Google Drive API, enable Google Sheets API.
+1.  Save the obtained JSON file into the `pyvecorg` package as `google_service_account.json`
+1.  Make sure it is ignored by Git
```

Comment:
I suppose the setup up until this point is needed for local hacking as well?

Reply:
Yes. But since it is decoupled, you should be able to run the site without members being populated, for editing other parts of the page.